modules/ROOT/pages/anypoint-platform-cli-commands.adoc

```diff
@@ -1205,6 +1206,43 @@ Besides the default `--help`, `-f`/`--fields` and `-o`/`--output` options, this
 | `--classifier` | Valid asset classifiers are `oas`, `raml`, `wsdl`, `http`, and `custom`.
 |===
 
+=== exchange asset uploadv2
+
+----
+> exchange asset uploadv2 [properties] <assetIdentifier>
+----
+
+This command uploads a rest-api, soap-api, http-api, custom, app, template, example, policy, or connector asset using the ID passed in <assetIdentifier>.
+
+Use the following format for the _assetIdentifier_ argument: `([_group_id_]/)_asset_id_/_version_. If _group_id_ is not specified, it defaults to the currently selected organization ID.
+
+[%header,cols="30a,40a,30a"]
+|===
+| Command | Description |  Example
+| `--name` (required) | Asset name .4+<.<|
+`exchange asset uploadv2 --name "Raml Asset" --description "RAML" --properties.apiVersion v1 --properties.mainFile api.raml --files.raml.zip /..../file-path/raml.zip raml-asset/1.0.0`
+
+`exchange asset uploadv2 --files.mule-application.jar /..../file-path/mule-app.jar --files.pom /..../file-path/mule-app.pom fa7b266c-3817-4cbb-ae49-7f3cc6c8cd9e/my-mule-app/1.0.0`
+    | `--name` (required)| Asset name 
+    | `--description` | Asset description
+    | `--type` | Asset type
+    
+    Required if no file is uploaded. If it is uploaded, type is inferred from the classifier of the file.
+    
+    Supported Values: `rest-api`, `soap-api`, `http-api`, `raml-fragment`, `custom`, `connector`, `template`, `example`, `policy`, `app`, `extension`, `external-library`
+    | `--keywords` | Keywords
+    | `--dependencies` | Asset dependencies, separated by ",". |`groupId:assetId:version,groupId2:assetId:version`
+    | `--properties` | Asset properties | `properties.mainFile main.yaml`
+    
+    `properties.apiVersion v1`
```

Comment:
The code formatting is not working on `properties.apiVersion v1` and `properties.assetLink \http://api.com` (I assume the intention was for them to look like `properties.mainFile main.yaml`)
![image](https://user-images.githubusercontent.com/62706868/117336892-131be780-ae73-11eb-84b4-8afe6d5f3f76.png)

Reply:
Fixed